plugin-server/src/capabilities.ts

```diff
@@ -83,6 +84,7 @@ export function getPluginServerCapabilities(config: PluginsServerConfig): Plugin
             return {
                 pluginScheduledTasks: true,
                 appManagementSingleton: true,
+                syncInlinePlugins: true,
```

Comment:
rebase and then add this to server.test.ts too where I now split things up in https://github.com/PostHog/posthog/pull/23601 and add it to functional-tests too